ui/app/templates/components/console/ui-panel.hbs

```diff
@@ -16,8 +16,9 @@
 <div class="console-ui-panel-content">
   <div class="content has-bottom-margin-l">
     <p class="console-ui-panel-intro is-font-mono has-bottom-margin-s">
-      The Vault Browser CLI provides an easy way to execute common Vault CLI commands, such as write, read, delete, and list.
-      It does not include kv v2 write or put commands. For guidance, type `help`.
+      The Vault Web REPL provides an easy way to execute common Vault CLI commands, such as write, read, delete, and list.
+      It does not include kv v2 write or put commands. For guidance, type `help`. For more detailed documentation, see the
```

Comment:
We recently made all references to KV v2 consistent like this, so let's go ahead and change that here too.
```suggestion
      It does not include KV version 2 write or put commands. For guidance, type `help`. For more detailed documentation, see the
```

Reply:
Oh nice - thanks!